docs/troubleshoot.md

```diff
@@ -2,6 +2,23 @@
 
 ## Trouble running the pytorch example
 
+### Segmentation fault using pytorch when reading dataset
+
+As reported in uber/petastorm#52, the reader appears haphazardly to dump core
+from segmentation fault.  @selitvin noted that these issues related to memory
+allocation have also been seen with TF.  A temporary workaround is to use a
+different memory allocator, such as `tcmalloc`, per below:
+
+```bash
+sudo apt-get install libtcmalloc-minimal4
+
+export LD_PRELOAD="/usr/lib/libtcmalloc_minimal.so.4" ; \
```

Comment:
I think the following line should do the trick. Does it?
```LD_PRELOAD="/usr/lib/libtcmalloc_minimal.so.4" python examples/mnist/pytorch_example.py```

Reply:
Sorry, Yevgeni. I don't mean to have you do work so late!

Preliminary test would indicate that this works.

I think the case where I found this not to work was with `pytest`, perhaps due to `LD_PRELOAD` not being effective in the sub-process environment without the export.